Find a functionally equivalent code:android.view.KeyEvent.keyCodeToString:COMMENT
Method Modifier: public      static      
Comment:/**
 * Returns a string that represents the symbolic name of the specified keycode
 * such as "KEYCODE_A", "KEYCODE_DPAD_UP", or an equivalent numeric constant
 * such as "1001" if unknown.
 *
 * @param keyCode The key code.
 * @return The symbolic name of the specified keycode.
 *
 * @see KeyCharacterMap#getDisplayLabel
 */

@@ -1,4 +1,4 @@
 {
-    String symbolicName = KEYCODE_SYMBOLIC_NAMES.get(keyCode);
-    return symbolicName != null ? symbolicName : Integer.toString(keyCode);
+    String symbolicName = nativeKeyCodeToString(keyCode);
+    return symbolicName != null ? LABEL_PREFIX + symbolicName : Integer.toString(keyCode);
 }

